nailgun/entities.py

```diff
@@ -2007,6 +2007,7 @@ class ContentCredential(
     """A representation of a Content Credential entity."""
 
     def __init__(self, server_config=None, **kwargs):
+        self.updatable_fields = ['name', 'content_type', 'content']
```

Comment:
This could be a class attribute, not an instance attribute, since its not defined on instantiation. Just a note though, works fine either way.

Reply:
Since we don't use class attributes anywhere, I think it's better to leave it as it is.